client/src/components/reviews/AddReview.jsx

```diff
@@ -254,6 +267,13 @@ class AddReview extends React.Component {
               <br></br>
               <input type='email' id='email' onChange={this.email.bind(this)} placeholder='Example: jackson11@email.com'></input>
             </div>
+            <div className='reviewFileUpload'>
+            <input type='file' onChange={this.upload.bind(this)} name='image' accept='image/png, image/jpeg'></input>
```

Comment:
I would suggest to bind the functions inside the constructor because constructor gets called only once per component so it creates a new object/function only once in bundle.js file